### PR TITLE
Upgrade keycloak from 24.0.5 to 25.0.1

### PR DESCRIPTION
**Pull request by akunzai**
- https://www.keycloak.org/docs/25.0.0/upgrading/

### CODE REVIEW

**Comment by akunzai** [comment]
cc @elkman

File: docs/release/1.4.14.txt [hunk 1]
```diff
@@ -1 +1 @@
-- keykloack version 24.0.5
+- keykloack version 25.0.1
```
Comment (elkman) [inline docs/release/1.4.14.txt:1]: should be moved to `docs/release/1.4.15.txt`

**Review by elkman — COMMENTED** [review]
Hi @akunzai, thanks for the PR. I've only had a very brief look at it, but it looks good.
I will need a few days to find more time and maybe involve a colleague to speed things up in the future.

File: keycloak-plugins/radius-disconnect-plugin/src/main/java/com/github/vzakharchenko/radius/dm/api/RadiusService.java [hunk 2]
```diff
@@ -2,7 +2,6 @@
 
 import com.github.vzakharchenko.radius.dm.models.RadiusInfoModel;
 import com.github.vzakharchenko.radius.dm.models.RadiusServiceModel;
-import org.jboss.resteasy.annotations.cache.NoCache;
```
Comment (nguyendangdtd) [inline keycloak-plugins/radius-disconnect-plugin/src/main/java/com/github/vzakharchenko/radius/dm/api/RadiusService.java:5]: could it replace by import org.jboss.resteasy.reactive.NoCache by adding below dependency
```
        <!-- https://mvnrepository.com/artifact/io.quarkus/quarkus-rest -->
        <dependency>
            <groupId>io.quarkus</groupId>
            <artifactId>quarkus-rest</artifactId>
            <version>3.12.0</version>
        </dependency>
```

Comment (nguyendangdtd) [reply keycloak-plugins/radius-disconnect-plugin/src/main/java/com/github/vzakharchenko/radius/dm/api/RadiusService.java:5]: The other thing is good for me.

Comment (akunzai) [reply keycloak-plugins/radius-disconnect-plugin/src/main/java/com/github/vzakharchenko/radius/dm/api/RadiusService.java:5]: Thanks, fixed

**Pull request merged**